tests/x509/test_x509_ext.py

```diff
@@ -1758,22 +1758,6 @@ def test_invalid_bit_string_padding_from_public_key(self, backend):
         with pytest.raises(ValueError, match="Invalid public key encoding"):
             _key_identifier_from_public_key(pretend_key)
 
-    def test_no_optional_params_allowed_from_public_key(self, backend):
```

Comment:
I didn't understand what this was trying to test _at all_, and now loading a DSA key with broken parameters is an error.

Reply:
Looks like we added it in 2019 to fix coverage of a conditional branch that no longer exists.

Reply:
Oh that would have been back with the old pure python DER parser.